src/test/java/scenarios/hooks/CampaignHooks.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala.
+ */
+package scenarios.hooks;
+
+import api.ApiRequestManager;
+import api.ApiResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.apache.http.HttpStatus;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import salesforce.endpointurl.Endpoints;
+import salesforce.entities.CreatedResponse;
+import salesforce.entities.Campaign;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CampaignHooks {
+    public Logger LOGGER = LogManager.getLogger(getClass());
```

Comment:
This logger is not being used. It should be remove it.

Reply:
Ok, I will add logger

Reply:
Ok, I will add logger